public/scripts/worktrollyUI.js

```diff
@@ -66,7 +66,7 @@ $("#Marketing").click(function() {
 });
 
 $("#backlogButton").click(function() {
-    selectedStatus = "Completed";
+    backlogSprint = "-1"
     newPage = "taskPage";
     uiLoader();
```

Comment:
When Backlog is clicked the page won't change rather, it will work as filter, We are just assigning the new backlog data into the dashboard.

Reply:
okay ok